packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -207,10 +222,54 @@ export function DifferentialAbundanceConfiguration(
     }
   }, [configuration, findEntityAndVariable]);
 
+  // If the variable is continuous, ask the backend for a list of bins
+  const continuousVariableBins = usePromise(
+    useCallback(async () => {
+      if (
+        !ContinuousVariableDataShape.is(
+          selectedComparatorVariable?.variable.dataShape
+        )
+      )
+        return;
+
+      const binRangeProps: GetBinRangesProps = {
+        studyId: studyMetadata.id,
+        ...configuration.comparator?.variable,
+        filters: filters ?? [],
+        dataClient,
+        binningMethod: 'quantile',
+      };
+      const bins = await getBinRanges(binRangeProps);
+      return bins;
+    }, [
+      dataClient,
+      configuration,
+      filters,
+      selectedComparatorVariable,
+      studyMetadata.id,
+    ])
+  );
+
   const disableSwapGroupValuesButton =
     !configuration?.comparator?.groupA && !configuration?.comparator?.groupB;
   const disableGroupValueSelectors = !configuration?.comparator?.variable;
 
+  // Create the options for groupA and groupB. Organizing into the LabeledRange[] format
+  // here in order to keep the later code clean.
+  const groupValueOptions = continuousVariableBins.value
+    ? continuousVariableBins.value.map((bin) => {
+        return {
+          min: bin.binStart,
+          max: bin.binEnd,
+          label: bin.binLabel,
+        } as LabeledRange;
+      })
+    : selectedComparatorVariable?.variable.vocabulary?.map((value) => {
+        return {
+          label: value,
+        } as LabeledRange;
+      });
```

Comment:
Instead of using a type cast, adding a return type to the callback function would be better. Type casts are more aggressive and can sometimes allow incorrect types to pass.
```suggestion
  const groupValueOptions = continuousVariableBins.value
    ? continuousVariableBins.value.map((bin): LabeledRange => {
        return {
          min: bin.binStart,
          max: bin.binEnd,
          label: bin.binLabel,
        };
      })
    : selectedComparatorVariable?.variable.vocabulary?.map((value): LabeledRange => {
        return {
          label: value,
        };
      });
```